### ISSUE TITLE
Import of blender camera property 'Shift X', 'Shift Y'.

### ISSUE THREAD

**Issue by lzyplayer** [xavier150/Blender-For-UnrealEngine-Addons#130]
Thanks for such great work!
I am currently using blender camera import part of this plugin. The transform and FOV part turns out well. However I am not sure camera shift is imported. Since camera shift help set cx cy of camera intrinsic, I assume this is a important feature.
```python
bpy.data.cameras["Camera"].shift_x
bpy.data.cameras["Camera"].shift_y
```
I haven't found any match of this property in source code of the plugin. 
Are these property imported? Does Unreal support camera shift? 
Thanks in advance for any reply!

**Comment by shared-iconem** [comment]
Hi, UE5 now have both an value SensorHorizontalOffset and SensorVerticalOffset (in Filmback). I think it should be a good option to add to automaticly convert blender sensor fit (horizontal or vertical) as well as sensor height and width and convert blender value of the shift to milimeters.

**Comment by xavier150** [comment]
Hello @shared-iconem 
I just checked and yes Unreal Engine 5.5 added camera shift support, I did not know that, I will add support for it in the next version for the addon.
Thanks for the suggestion and the info!

Btw I have a page with some details about it: https://github.com/xavier150/Blender-For-UnrealEngine-Addons/wiki/Camera-Shift

**Comment by xavier150** [comment]
Hello @lzyplayer @shared-iconem 
I updated the addon to support camera shift and sensor offset now that it possible.

I pushed the dev branch if you want to try it now:
Dev branch: https://github.com/xavier150/Blender-For-UnrealEngine-Addons/wiki/Download-And-Installation-From-Dev-Branch
Thanks for the suggestion!

- New: Native support for Unreal Engine 5.5 camera sensor offset (shift_x and shift_y).

I will include the change in the next version of the addon.
And close this issue when the version is released.
